docs/source/tutorial/understanding_evaluation.rst

```diff
@@ -0,0 +1,11 @@
+Understanding the Evaluation
+============================
+This part of the tutorial is aimed to help you understand the rank-based evaluation metrics reported
+in :class:`pykeen.evaluation.RankBasedMetricResults`.
+
+Side-Specific Metrics
```

Comment:
@mberr fill out here, please

Reply:
Since there is no general discussion of the evaluation, it might be better to start there :wink: I can take care of it, but it will take me a bit :slightly_smiling_face:

Reply:
Since this PR is not a fix, but a new feature, I think it should be okay to have some delay here to improve the documentation :slightly_smiling_face:

Reply:
@cthoyt I started adding a bit about the general evaluation :slightly_smiling_face: